src/main/java/org/opentripplanner/netex/loader/mapping/NetexMapper.java

```diff
@@ -0,0 +1,170 @@
+package org.opentripplanner.netex.loader.mapping;
+
+import com.google.common.collect.Multimap;
+import org.opentripplanner.model.Agency;
+import org.opentripplanner.model.Notice;
+import org.opentripplanner.model.NoticeAssignable;
+import org.opentripplanner.model.Route;
+import org.opentripplanner.model.ServiceCalendarDate;
+import org.opentripplanner.model.Stop;
+import org.opentripplanner.model.StopTime;
+import org.opentripplanner.model.Trip;
+import org.opentripplanner.model.impl.OtpTransitServiceBuilder;
+import org.opentripplanner.netex.loader.NetexImportDataIndex;
+import org.opentripplanner.netex.loader.NetexImportDataIndexReadOnlyView;
+import org.opentripplanner.netex.support.DayTypeRefsToServiceIdAdapter;
+import org.opentripplanner.routing.edgetype.TripPattern;
+import org.rutebanken.netex.model.Authority;
+import org.rutebanken.netex.model.JourneyPattern;
+import org.rutebanken.netex.model.Line;
+import org.rutebanken.netex.model.NoticeAssignment;
+import org.rutebanken.netex.model.StopPlace;
+
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+
+// TODO OTP2 - Add Unit tests
+
+/**
+ * <p>
+ * This is the ROOT mapper to map from the Netex domin model into the OTP internal model. This class delegates to
+ * type/argegate specific mappers and take the result from each such mapper and add the result to the
+ * {@link OtpTransitServiceBuilder}.
+ * </p>
+ * <p>
+ * The transit builder is updated with the new OTP model entities, holding ALL entities parsed so fare including
+ * previous Netex files in the same bundle. This enable the mapping code to make direct references between entities
+ * in the OTP domain model.
+ * </p>
+ */
+public class NetexMapper {
```

Comment:
Not sure way GitHub believe this is a new class, this was moved from `org.opentripplanner.netex.mapping` to `org.opentripplanner.netex.loader.mapping` - this apply to most mappers.

I have updated this class so it now is the only place setting the result of the mapping into the transitBuilder. The "sub" mappers do not alter state - they return their result. I enforced this by passing in ReadOnly data structures to the sub-mappers.